api/users.js

```diff
@@ -190,3 +196,40 @@ router.patch("/me/password", authenticate, async (req, res, next) => {
     next(e);
   }
 });
+
+//GET logged in user interests for suggestion component
+router.get("/me/interests", authenticate, async (req, res, next) => {
+  try {
+    const myInterests = prisma.user.findUniqueOrThrow({
+      where: { id: req.user.id },
+      include: {
+        interests: true,
+      },
+    });
+    res.json(myInterests);
```

Comment:
I think we also wouldn't have to authenticate "/me/interests" - since we're saying that any user can access anyone else's interests too